src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/Microsoft.NET.Build.Tasks.UnitTests.csproj

```diff
@@ -38,4 +38,8 @@
 
   <Import Project="Sdk.targets" Sdk="RoslynTools.RepoToolset" />
 
+  <!-- Workaround for fixed bug causes duplicate file warnings on pack, so disable it
+       https://github.com/dotnet/roslyn-tools/issues/249 -->
+  <Target Name="WorkaroundIncorrectSatelliteDeployment" />
```

Comment:
You're going to get the repo toolset fix in before this, right?

Reply:
@nguerrera I've merged https://github.com/dotnet/roslyn-tools/pull/253, and updated this PR to use the new repo toolset and remove the workaround.